tests/framework/installer/installer.go

```diff
@@ -31,8 +31,8 @@ import (
 const (
 	// Version tag for the latest manifests
 	VersionMaster = "master"
-	// Version tag for Rook v0.8
-	Version0_8 = "v0.8.3"
+	// Version tag for Rook v0.9
+	Version0_9 = "v0.9.1"
```

Comment:
do you have good ideas about the process to use to keep this up to date as we do patch releases?

Reply:
good question. The best thing I can think of is to add it to the checklist to update in master after a patch release.